core/dbt/include/global_project/macros/materializations/models/incremental/incremental.sql

```diff
@@ -6,11 +6,12 @@
   {% set target_relation = this.incorporate(type='table') %}
   {% set existing_relation = load_relation(this) %}
   {% set tmp_relation = make_temp_relation(target_relation) %}
+  {%- set tmp_identifier = tmp_relation['identifier'] -%}
```

Comment:
The use of `tmp_idenfier` below is to create the `intermediate_relation` below, which is actually the exact same as `tmp_relation`. We may be able to delete some lines of code, which is always a cool feeling

Reply:
Cleaned this up as well 🧹